server/src/routes/rootRouter.js

```diff
@@ -1,8 +1,9 @@
 import express from "express";
 
+import { squidsRouter } from "./api/v1/squidsRouter.js";
 import { clientRouter } from "./clientRouter.js";
 
-const rootRouter = new express.Router();
-rootRouter.use("/", clientRouter);
+export const rootRouter = new express.Router();
 
-export { rootRouter };
+rootRouter.use("/", clientRouter);
+rootRouter.use("/api/v1/squids", squidsRouter);
```

Comment:
It would be good to extract out an api router, and a v1 router, and then have the v1 router import the squids router (I think I'm the one who set up a single .use for "/api/v1/squids" while we were debugging, so that's on me :) )